phoenix-core/src/it/java/org/apache/phoenix/end2end/index/ViewIndexIT.java

```diff
@@ -153,15 +153,10 @@ public void testDeleteViewIndexSequences() throws Exception {
         conn2.createStatement().executeQuery("SELECT * FROM " + fullTableName).next();
         String sequenceName = getViewIndexSequenceName(PNameFactory.newName(fullTableName), null, isNamespaceMapped);
         String sequenceSchemaName = getViewIndexSequenceSchemaName(PNameFactory.newName(fullTableName), isNamespaceMapped);
-        String seqName = getViewIndexSequenceName(PNameFactory.newName(fullTableName), null, !isNamespaceMapped);
-        String seqSchemaName = getViewIndexSequenceSchemaName(PNameFactory.newName(fullTableName), !isNamespaceMapped);
         verifySequenceValue(null, sequenceName, sequenceSchemaName, -9223372036854775807L);
         verifySequenceValue(null, sequenceName, sequenceSchemaName, -9223372036854775807L);
         conn1.createStatement().execute("CREATE INDEX " + indexName + "_2 ON " + fullViewName + " (v1)");
         verifySequenceValue(null, sequenceName, sequenceSchemaName, -9223372036854775806L);
```

Comment:
The test is verifying that a CREATE INDEX statement on a view will increment the sequence. And yes, it would be more clear with a constant, because they're offsets of Long.MIN_VALUE.